cmd/utils/flags.go

```diff
@@ -1008,14 +1008,14 @@ func setDataDir(cfg *node.Config) {
 		cfg.DataDir = viper.GetString(DataDirFlag.Name)
 	case environment == params.DevName:
 		cfg.DataDir = "" // unless explicitly requested, use memory databases
-	case environment == params.GardenName && cfg.DataDir == node.DefaultDataDir():
-		cfg.DataDir = filepath.Join(node.DefaultDataDir(), params.GardenName)
-	case environment == params.OrchardName && cfg.DataDir == node.DefaultDataDir():
-		cfg.DataDir = filepath.Join(node.DefaultDataDir(), params.OrchardName)
-	case environment == params.LighthouseName && cfg.DataDir == node.DefaultDataDir():
-		cfg.DataDir = filepath.Join(node.DefaultDataDir(), params.LighthouseName)
-	case environment == params.LocalName && cfg.DataDir == node.DefaultDataDir():
-		cfg.DataDir = filepath.Join(node.DefaultDataDir(), params.LocalName)
+	case environment == params.GardenName && cfg.DataDir == xdg.DataHome:
```

Comment:
What is this?

Reply:
This is just rebase related. It will drop off when its rebased to main